classes/class-cpt-wplf-form.php

```diff
@@ -522,7 +522,10 @@ function maybe_enqueue_frontend_script() {
 
     if( is_a( $post, 'WP_Post' ) && ( has_shortcode( $post->post_content, 'libre-form') || $post->post_type === 'wplf-form') ) {
       wp_enqueue_script( 'wplf-form-js' );
-      wp_localize_script( 'wplf-form-js', 'ajax_object', array( 'ajax_url' => admin_url( 'admin-ajax.php' ) ) );
+      wp_localize_script( 'wplf-form-js', 'ajax_object', array( 
+        'ajax_url' => admin_url( 'admin-ajax.php' ),
+        'ajax_credentials' => apply_filters('wplf_frontend_script_credentials', 'same-origin')
```

Comment:
Ah, the default is defined here.

Not sure about the naming for this filter. It should at least have "fetch" somewhere in it.

How about: `wplf_ajax_fetch_credentials_mode` ?